### PR TITLE
npm removed

### PR DESCRIPTION
**Pull request by Muhammad-Ali-Ahmad**

### CODE REVIEW

File: app/views/main.scala.html [hunk 1]
```diff
@@ -208,12 +208,7 @@
         "googleAnalytics" -> googleAnalytics,
         "gaCustomEvent" -> gaCustomEvent,
         "scriptElems" -> Seq(
-            Map("url" -> controllers.routes.AssetsController.versioned("javascript/tai-new.js")),
-            Map("url" -> controllers.routes.AssetsController.versioned("javascript/tai-backlink.js")),
-            Map("url" -> controllers.routes.AssetsController.versioned("javascript/char-count.js")),
-            Map("url" -> controllers.routes.AssetsController.versioned("javascript/card.js")),
-            Map("url" -> controllers.routes.AssetsController.versioned("javascript/tax-code-change.js")),
-            Map("url" -> controllers.routes.AssetsController.versioned("javascript/urbanner.js"))
+            Map("url" -> controllers.routes.AssetsController.versioned("javascripts/tai-app.js"))
```
Comment (struong) [inline app/views/main.scala.html:211]: same change(s) required in this main to the "other" main views e.g. `https://github.com/hmrc/tai-frontend/blob/master/app/views/main_without_nav.scala.html`

**Review by struong — APPROVED** [review]

File: app/views/main.scala.html (hunk 1 above)
Comment (Muhammad-Ali-Ahmad) [reply app/views/main.scala.html:211]: done

**Review by timsb — APPROVED** [review]
Looks good to me. The only other thing I can think you may want to consider adding is that you can speed up the assets compilation by specifying to use NodeJS (if you have it). Info about it at https://www.playframework.com/documentation/2.8.x/AssetsOverview#Managed-assets which is pretty much just adding `export SBT_OPTS="$SBT_OPTS -Dsbt.jse.engineType=Node"`. Not 100% necessary but something you could put in the README.md

**Comment by Muhammad-Ali-Ahmad** [comment]
Thanks @timsb. I have added that in readme

File: .gitignore [hunk 2]
```diff
@@ -18,7 +18,6 @@ lib
 lib_managed
 log
 logs
-modules
```
Comment (struong) [inline .gitignore:21]: leave these in for now, it does no harm and will prevent anyone with an existing repo from pushing these up accidentally

File: project/plugins.sbt [hunk 3]
```diff
@@ -14,4 +14,12 @@ addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "2.13.0")
...
\ No newline at end of file
+addSbtPlugin("com.lucidchart" % "sbt-scalafmt" % "1.16")
+
+addSbtPlugin("org.irundaia.sbt" % "sbt-sassify" % "1.5.1")
+
+addSbtPlugin("net.ground5hark.sbt" % "sbt-concat" % "0.2.0")
+
+addSbtPlugin("com.typesafe.sbt" % "sbt-uglify" % "2.0.0")
+
+addSbtPlugin("com.typesafe.sbt" % "sbt-digest" % "1.1.4")
```
Comment (struong) [inline project/plugins.sbt:25]: don't think we need this anymore?

**Review by struong — APPROVED** [review]

**Review by struong — APPROVED** [review]

**Pull request closed without merging**